.github/workflows/ci.yaml

```diff
@@ -95,6 +97,7 @@ jobs:
         if [ "$NEWSHELL" == "true" ]; then
           export R2_CFG_NEWSHELL=1
         fi
+        pip3 install --user 'git+https://github.com/radareorg/radare2-r2pipe#egg=r2pipe&subdirectory=python'
```

Comment:
install this in the step `Install meson and ninja` and rename it to `install python dependencies` or something like that.

Reply:
can't combine the steps because the mac build doesn't require meson, but i've moved the r2pipe installation up 4dbe23b65300a946db7e1724dee8702386b2297e